test/integration/targets/ansible-config/tasks/main.yml

```diff
@@ -56,3 +56,76 @@
       that:
           - valid_env is success
           - invalid_env is failed
+
+
+- name: dump galaxy_server config
+  environment:
+    ANSIBLE_CONFIG: '{{ role_path }}/files/galaxy_server.ini'
+  vars:
+    expected:
+      my_org_hub:
+        url:
+          value: "https://automation.my_org/"
+          origin: role_path ~ "/files/galaxy_server.ini"
+        username:
+          value: my_user
+          origin: role_path ~ "/files/galaxy_server.ini"
+        password:
+          value: my_pass
+          origin: role_path ~ "/files/galaxy_server.ini"
+        api_version:
+          value: None
+          origin: default
+      release_galaxy:
+      test_galaxy:
+      my_galaxy_ng:
+    gen_keys: '{{expected.keys()|map("regex_replace", "^.+$", "GALAXY_SERVER.\1") }}'
+  block:
+  - ansible.builtin.command: ansible-config dump --type {{ item }} --format json
+    loop:
+    - base
+    - all
+    register: galaxy_server_dump
+
+#  - set_fact:
```

Comment:
Are you planning on re-enabling these tests?

Reply:
yes, was thinking of moving the servers to dict with k=v subdicts to do this and more tests